purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -870,6 +872,79 @@ class PurchasesTest {
         assertThat(receivedUserCancelled).isFalse()
     }
 
+    @Test
+    fun `when purchasing a product with multiple purchase options, we choose the default`() {
+        val productId = "onemonth_freetrial"
+
+        val basePlanPurchaseOption = stubPurchaseOption("base-plan-purchase-option")
+        val freeTrialPurchaseOption = stubPurchaseOption(
+            "free-trial-purchase-option",
+            pricingPhases = listOf(stubFreeTrialPricingPhase(), stubPricingPhase())
+        )
+        val storeProduct = stubStoreProduct(
+            productId = productId,
+            defaultOption = freeTrialPurchaseOption,
```

Comment:
nit: maybe we can make this even more explicit/readable by naming the free trial "expectedDefaultPurchaseOption"?